packages/agent/src/agents/core/snakAgent.ts

```
@@ -6,6 +6,8 @@ import { BaseMessage, HumanMessage, AIMessage } from '@langchain/core/messages';
 import { DatabaseCredentials } from '../../tools/types/database.js';
 import { AgentMode, AGENT_MODES } from '../../config/agentConfig.js';
 import { MemoryConfig } from '../operators/memoryAgent.js';
+import { CustomHuggingFaceEmbeddings } from '@snakagent/core';
```

ESLint

[error] 'CustomHuggingFaceEmbeddings' is defined but never used.

(@typescript-eslint/no-unused-vars)